Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLMeshObject.cs

```diff
@@ -176,6 +205,34 @@ protected void UploadToRGL()
                 }
             }
         }
+
+        public void UploadUVs()
+        {
+            Vector2[] UVs = Mesh.uv;
+            bool uvOK = UVs != null && UVs.Length > 0;
+
+            unsafe
+            {
+                if(uvOK)
```

Comment:
Inverse condition and move above unsafe context. Throw an exception if uv not OK.